### DISCUSSION TITLE
[INFRA/eyes][Feature] support github action cache for docker images

### DISCUSSION THREAD

**Discussion by xxchan** [apache/skywalking#10428] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Every run of skywalking-eyes takes 1min to build the docker image. I guess GHA cache can be utilized to speedup that...

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
@xxchan If you are not going to submit a pull request for this, I think this is not a planned feature for maintainers. I am going to move this to discussion. If you want contribute, please let me know.

**Comment by xxchan** [comment]
Ok please move to discussion. I'm willing to do that but not recently.

**Comment by kezhenxu94** [comment]
@xxchan I filed a [PR](https://github.com/apache/skywalking-eyes/pull/149) to leverage the new `composite` mode of GHA and set up caches for the build, no Docker is needed and the overall time looks like this

- first run (~24s)

<img width="1688" alt="image" src="https://user-images.githubusercontent.com/15965696/221078401-2df04c07-74b0-4973-89c6-b5754a40f005.png">

- non first run (~4s)

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/15965696/221078268-f9eda3c2-15fc-40ff-ba5f-27a762c8c708.png">


@xxchan wanna try and give some suggestions?
